core/src/main/java/org/mskcc/cbio/portal/model/GenesetInfo.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015 Memorial Sloan-Kettering Cancer Center.
```

Comment:
this diff is because DaoGeneSet was renamed/replaced by DaoGeneset. And GenesetInfo is new...and they are mostly similar (% 60 ) so git thinks it should label this as a rename.